### DISCUSSION TITLE
Explanation

### DISCUSSION THREAD

**Discussion by alfille** [free-audio/clap#87] [General]
I got to Clap from the Linux Weekly article. Both that article and especially the README here give no explanation of what Clap does. 

It's easy to be so immersed in a project that a general overview for a curious if naive audience is overlooked.

Would you consider adding an explanation as a first paragraph to the README?

**Comment by abique** [comment]
Hi,
That was fast! :-)
So CLAP is an audio plugin API, similar to VST2/3, AU, LV2, ...
I'll check the readme.

**Reply by abique** [reply]
I've updated the documentation.
